src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -285,7 +285,7 @@ internal PackageManagerClientViewModel(DynamoViewModel dynamoViewModel, PackageM
 
             if (AuthenticationManager.LoginState.Equals(LoginState.LoggedIn))
             {
-                this.Uservotes = this.Model.UserVotes();
+                Task.Run(() => this.Uservotes = this.Model.UserVotes());                
```

Comment:
this is going to execute from other some other thread pool thread. 
1. Does anything else write to `UserVotes`? (another thread?) - it seems like it could be made readonly if not.

Reply:
Hi, sorry for the late reply on this thread! 

I am not sure if I got this correctly - the UserVotes() returns the results of the Greg call:

![image](https://github.com/DynamoDS/Dynamo/assets/5354594/83ba98a5-1057-484f-a666-4b2abc1937be)

It is only used once when we create the `PackageManagerClientViewModel` and fetches any votes from the current user, I think that's about it.

Reply:
I say keep it in a Task.Run, but move it closer to the greg client call.
Having it in the VIewModel is risky, because someone might add UI code (code that interacts with UI elements) in the UserVotes body
As @mjkkirschner mentioned, we need to make sure all data that is accessed by other thread is thread safe (in some way or another)
ALso we need to make sure we handle exceptions properly. If ExecuteAndDeserialize throws an exception, what happens?

Reply:
I think I got it. I moved the call before using the `Uservotes` for the first time.